metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -541,6 +559,57 @@ static void validateNewTopicNames(Map<String, ApiError> topicErrors,
         return configChanges;
     }
 
+    Map<String, ResultOrError<Uuid>> findTopicIds(long offset, Collection<String> names) {
+        Map<String, ResultOrError<Uuid>> results = new HashMap<>();
+        for (String name : names) {
+            if (name == null) {
+                results.put(null, new ResultOrError<>(
+                    new ApiError(INVALID_REQUEST, "Invalid null topic name.")));
```

Comment:
This can be replaced by the constructor `public ResultOrError(Errors error, String message)`